### PR TITLE
Feat/column data type polars types

### PR DESCRIPTION
**Pull request by philter87**
## Description

- This PR adds polars_types to ColumnDataType. We need a mapping between the polars types and "Sprout" types as we use polars.read_csv to find the data types of the CSV columns. I am creating this small PR to avoid a big PR. 
- The method 'ColumnDataType.get_from_series()' returns a ColumnDataType based on the series.dtype

## Related Issues

See also Issues #184

## Testing

- [X] Yes
- [ ] No, not needed (give a reason below)
- [ ] No, I need help writing them

## Reviewer Focus
This PR only needs a quick review.

## Checklist

For all PRs that are not general documentation

- [X] Tests accompany or reflect changes to the code
- [X] Tests ran and passed locally
- [X] Ran the linter and formatter
- [X] Build has passed locally
- [X] Relevant documentation has been updated

### CODE REVIEW

File: app/models/column_data_type.py [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
     display_name = models.TextField()
     description = models.TextField()
+    polars_types = models.TextField(default="")
+
+    @staticmethod
+    def get_from_series(series: Series) -> "ColumnDataType":
+        """Finds ColumnDataType based on series dtype.
+
+        Args:
+            series: The polars.Series to find ColumnDataType from
```
Comment (lwjohnst86) [inline app/models/column_data_type.py:18]:
```suggestion
            series: The polars.Series to find the  ColumnDataType from
```

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Love it! 😁  I only have a small suggestion to the text, since it seems like an incomplete sentence, but I wasn't sure.

**Review by K-Beicher — DISMISSED** [review]
Not that I'm understanding everything, but it looks good from what I can see.

**Review by K-Beicher — DISMISSED** [review]
Looks good

**Review by lwjohnst86 — APPROVED** [review]
Neat!! Only thing I wonder about, though it is low priority, is the use of the for loop (still intuitively don't love them), the use of `@staticmethod`. But otherwise, great!

**Review by K-Beicher — APPROVED** [review]
Cool

**Review by signekb — APPROVED** [review]
Nice 👍

**Pull request merged**